src/Console/Output/Progress/PercentageBarOutput.php

```diff
@@ -35,9 +35,9 @@ public function __construct(OutputContext $context)
         $this->context = $context;
 
         $this->progressBar = new ProgressBar($context->getOutput(), $this->context->getFilesCount());
-        $this->progressBar->setBarCharacter('█');
-        $this->progressBar->setEmptyBarCharacter('░');
-        $this->progressBar->setProgressCharacter('░');
+        $this->progressBar->setBarCharacter('▓'); // dark shade character \u2593
+        $this->progressBar->setEmptyBarCharacter('░'); // light shade character \u2591
+        $this->progressBar->setProgressCharacter('');
         $this->progressBar->setFormat('normal');
```

Comment:
Can we use `SymfonyStyle::createProgressBar()` directly?

Reply:
I thought about it: it requires non-trivial changes to propagate all the necessary required dependencies.

I guessed @Wirone already knew that and chose the way this class is written for that reason.

Reply:
I just wanted to have stable UX, not dependant on external style that can be changed at any point. Did not even verify if this would be more complicated, just configured how it should look like 😅. This PR makes sense of course, even though I chose lighter character on purpose 😉.